packages/gasket-preset-nextjs/EXAMPLES.md

```diff
@@ -0,0 +1,384 @@
+# @gasket/preset-nextjs Examples
+
+This document provides examples of using the `@gasket/preset-nextjs` preset and understanding its configurations.
+
+## Using the Preset
+
+### Creating a new Next.js project
+
+```bash
+# Create a new Next.js project using the preset
+npx create-gasket-app@latest my-app --presets @gasket/preset-nextjs
+
+# Or with yarn
+yarn create gasket-app my-app --presets @gasket/preset-nextjs
+
+# With config to skip prompts
+npx create-gasket-app@latest my-app \
+  --presets @gasket/preset-nextjs \
+  --config '{"typescript": true, "nextServerType": "appRouter", "nextDevProxy": false}'
+```
+
+## Generated Project Structure
+
+### JavaScript Project (App Router)
+
+```
+my-app/
+├── app/
+│   ├── layout.js          # Root layout component
+│   └── page.js            # Home page component
+├── next.config.js         # Next.js configuration
+├── gasket.js              # Gasket configuration
+├── package.json           # Project configuration
+└── README.md              # Project documentation
+```
+
+### TypeScript Project (Custom Server)
+
+```
+my-app/
+├── app/
+│   ├── layout.tsx         # Root layout component (TypeScript)
+│   └── page.tsx           # Home page component (TypeScript)
+├── server.ts              # Custom server entry point
+├── next.config.js         # Next.js configuration
+├── gasket.ts              # Gasket configuration (TypeScript)
+├── tsconfig.json          # TypeScript configuration
+├── package.json           # Project configuration
+└── README.md              # Project documentation
+```
+
+## Server Type Examples
```

Comment:
It seems like these sections with all the file contents is too verbose and may prove difficult to maintain. I'm not sure it is actually useful either.

Optionally reduce or remove.